custom_components/powercalc/data/yeelight/YLDL01YL/ambilight/model.json

```diff
@@ -0,0 +1,3 @@
+{
+    "standby_power": 0.72,
```

Comment:
could you remove the extra , in the end. The JSON is invalid now.

Reply:
Fixed